staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresourcedefinition/etcd.go

```diff
@@ -223,7 +223,7 @@ func (r *REST) Delete(ctx context.Context, name string, deleteValidation rest.Va
 
 	crd := obj.(*apiextensions.CustomResourceDefinition)
 	if IsCrdSystemForced(crd) && tenant != metav1.TenantSystem {
```

Comment:
what does "Forced" mean here?

Reply:
Doc https://github.com/CentaurusInfra/arktos/blob/master/docs/design-proposals/multi-tenancy/multi-tenancy-api-resource-model.md gives the defintion of "System Forced" as below：
“When a system CRD is usable to all regular tenants, this sharing can be:

forced. It overrides the local CRDs with the same name in non-system tenant spaces. Once a forced system CRD is deployed, a regular tenant's attempt to create a CRD with the same name will be rejected."

Reply:
off topic but seems forced is a bit hard choice of word. maybe "overridden" or something.